shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -125,6 +126,36 @@ void setAccessibilityEnabled(boolean accessibilityEnabled) {
         }
     }
 
+    private boolean shouldSetCollectionInfo(SemanticsObject object) {
+        // TODO(dnfield): make these lambdas when Java 1.8 support lands.
+        Predicate<SemanticsObject> parentMatcher = new Predicate<SemanticsObject>() {
+            @Override
+            public boolean test(SemanticsObject o) {
+                return o == mA11yFocusedObject;
+            }
+        };
+
+        Predicate<SemanticsObject> flagMatcher = new Predicate<SemanticsObject>() {
+            @Override
+            public boolean test(SemanticsObject o) {
+                return o.hasFlag(Flag.HAS_IMPLICIT_SCROLLING);
+            }
+        };
+
+        // TalkBack expects a number of rows and/or columns greater than 0 to announce
+        // in list and out of list.  For an infinite or growing list, you have to
+        // specify something > 0 to get "in list" announcements.
+        // TalkBack will also only track one list at a time, so we only want to set this
+        // for a list that contians the current a11y focused object - otherwise, if there
+        // are two lists or nested lists, we may end up with announcements for only the last
+        // one that is currently availalbe in the semantics tree.  However, we also want
```

Comment:
typo: availalbe -> available
```suggestion
        // one that is currently available in the semantics tree.  However, we also want
```